bbq/vm/test/vm_test.go

```diff
@@ -4353,3 +4353,79 @@ func TestBlockScope2(t *testing.T) {
 		require.Equal(t, vm.NewIntValue(4), actual)
 	})
 }
+
+func TestIntegers(t *testing.T) {
+
+	t.Parallel()
+
+	test := func(integerType sema.Type) {
+
+		t.Run(integerType.String(), func(t *testing.T) {
+
+			t.Parallel()
+
+			result, err := compileAndInvoke(t,
+				fmt.Sprintf(`
+                        fun test(): %s {
+                            return 2 + 3
+                        }
+                    `,
+					integerType,
+				),
+				"test",
+			)
+			require.NoError(t, err)
+
+			assert.Equal(t, vm.NewIntValue(5), result)
+		})
+	}
+
+	// TODO:
```

Comment:
Future TODO: We should enable this once the VM has support for all integer constants and values